xarray/core/dataset.py

```diff
@@ -5211,6 +5392,56 @@ def drop_vars(
             passed are not in the dataset. If 'ignore', any given names that are in the
             dataset are dropped and no error is raised.
 
+        Examples
+        --------
+
+        >>> dataset = xr.Dataset(
```

Comment:
You've made a dataset with 3 variables, then dropped one of them. You may as well show that it's possible to drop two at once also.

Also show what `errors` does.

Reply:
How do I mention the Raise value error as output?

Reply:
Good question. I assume you just copy and paste the output you get when the error is raised, but you should try it out and see if it passes the doctests. Might want to google it first too to check.